### ISSUE TITLE
What would cause "Sort Videos" to fail? (add videos, save, re-open, no videos in list)

### ISSUE THREAD

**Issue by walt93** [WWBN/AVideo#9687]
**Describe the bug**
See title

**To Reproduce**
1. Go to My Videos
2. Click on Sort Videos
3. Add some videos
4. Save
5. Close
6. Re-open
7. Observe there are no videos in the list

**Expected behavior**
The videos I added should remain in the list

**Error Logs**
Provide the log files at the moment you found the issue.
If you are not sure how to find the log file, please check here https://github.com/WWBN/AVideo/wiki/How-to-find-errors-on-AVideo-Platform

**Screenshots**

https://github.com/user-attachments/assets/b021db79-375f-4c5e-8c2b-925d8276684c

**Comment by eitanwa** [comment]
I just discovered this amazing option recently
It is very excellent for controlling the last uploads
It will do the trick

![image](https://github.com/user-attachments/assets/cd07da66-63af-45f6-ac26-fd2d0b2e57bc)

**Comment by walt93** [comment]
Correct. That is the button that I pressed to add the videos to the list. I have tested the flow several times now. Each time after adding and saving, when you return, there are no items in the list.

**Comment by eitanwa** [comment]
Okay

However, it does work for us @DanielnetoDotCom 
What I would like to know is why there are 2 saves?
Both (on our site) do the same action
